### PR TITLE
Feature/vudrm 1747/bento4 docs

### PR DESCRIPTION
**Pull request by bengo1990**
Bento4 docs.

https://docs.vualto.com/projects/vudrm/en/feature-vudrm-1747-bento4-docs/DeveloperDocumentation/PackagingGuides/bento4.html

### CODE REVIEW

**Review by somerobmitchell — DISMISSED** [review]

File: docs/source/DeveloperDocumentation/PackagingGuides/bento4.md [hunk 1]
```diff
@@ -1 +1,110 @@
...
+            <cpix:HLSSignalingData playlist="media"><media_playlist_value></cpix:HLSSignalingData>
+        </cpix:DRMSystem>
+    </cpix:DRMSystemList>
+    <cpix:ContentKeyUsageRuleList></cpix:ContentKeyUsageRuleList>
+</cpix:CPIX>
+```
+
+#### CPIX keys as JSON
+
+Below is an example of a CPIX keys as JSON document with all `DRM Providers` selected.
```
[the remark this answers is not in the document]
Comment (bengo1990) [reply docs/source/DeveloperDocumentation/PackagingGuides/bento4.md:63]: Actioned.

File: docs/source/DeveloperDocumentation/PackagingGuides/bento4.md [hunk 2]
```diff
@@ -1 +1,110 @@
...
+A guide can be found at the following link - [VUDRM Admin - CPIX Document Request](https://docs.vualto.com/projects/vudrm/en/latest/UserGuide/VUDRM-Admin.html#vudrm-encryption-keys)
+
+### Examples of Each Document
+
+#### CPIX Document
+
+Each DRM Provider has a unique `systemId`. This helps to distinguish which values belong to which DRM Provider in a CPIX Document. 
+
+```
+System IDs:
```
[the remark this answers is not in the document]
Comment (bengo1990) [reply docs/source/DeveloperDocumentation/PackagingGuides/bento4.md]: Actioned.

**Review by somerobmitchell — APPROVED** [review]

File: docs/source/DeveloperDocumentation/PackagingGuides/bento4.md [hunk 3]
```diff
@@ -1 +1,84 @@
-# Bento4
\ No newline at end of file
+# Bento4
+
+## Installing Bento4
+
+You can download the binaries for Bento4 for your preferred OS at the following link - [Bento4 binaries](https://www.bento4.com/downloads/)
```
Comment (PerritonBranch) [inline docs/source/DeveloperDocumentation/PackagingGuides/bento4.md:5]: General website standards, from places like [w3](https://www.w3.org/QA/Tips/noClickHere#:~:text=Don't%20use%20%22click%20here%22%20as%20link%20text&text=When%20calling%20the%20user%20to,doesn't%20talk%20about%20mechanics), are against linking like this.
Something like the follow would be nicer:
```suggestion
You can download the binaries for your preferred OS from [Bento4](https://www.bento4.com/downloads/).
```

File: docs/source/DeveloperDocumentation/PackagingGuides/bento4.md [hunk 4]
```diff
@@ -1 +1,84 @@
...
+
+Add the filepath for the binaries to the Bento4 `bin` folder as a value to your `Path` environemnt variables on your computer.
+
+## Get CPIX Documents
+
+CPIX Documents are required to give Bento4 the necessary information in order to encrypt your media.
+
+You can retrieve these documents from the VUDRM Admin site. 
+
+A guide can be found at the following link - [VUDRM Admin - CPIX Document Request](https://docs.vualto.com/projects/vudrm/en/latest/UserGuide/VUDRM-Admin.html#vudrm-encryption-keys)
```
Comment (PerritonBranch) [inline docs/source/DeveloperDocumentation/PackagingGuides/bento4.md:15]: As above.

File: docs/source/DeveloperDocumentation/PackagingGuides/bento4.md [hunk 5]
```diff
@@ -1 +1,84 @@
...
+
+More information on our CPIX Key Provider API can be found at the following link - [CPIX Key Provider API](https://docs.vualto.com/projects/vudrm/en/latest/DeveloperDocumentation/VUDRM-key-provision.html#cpix-key-provider-api)
```
Comment (PerritonBranch) [inline docs/source/DeveloperDocumentation/PackagingGuides/bento4.md:17]: And again.

File: docs/source/DeveloperDocumentation/PackagingGuides/bento4.md [hunk 6]
```diff
@@ -1 +1,84 @@
...
+</cpix:CPIX>
+```
+
+## Package your content
+
+Now you have retrieved the relevant documentation, you can package content with Bento4.
+
+The values from the CPIX documents will need to be copied and pasted to the corresponding section in the examples below.
+
+For more information, please refer to the Bento4 Documentation which can be found at the following link - [Bento4 Documentation](https://www.bento4.com/documentation/mp4dash/)
```
Comment (PerritonBranch) [inline docs/source/DeveloperDocumentation/PackagingGuides/bento4.md:60]: Okay gunna stop pointing these out now but you get the idea.

File: docs/source/DeveloperDocumentation/PackagingGuides/bento4.md [hunk 7]
```diff
@@ -1 +1,84 @@
...
+
+### Packaging Content with Widevine and Playready DRM
+
+Below is an example of how we can send a request to Bento4 to encrypt our content with Widevine and Playready DRM and package it:
```
Comment (PerritonBranch) [inline docs/source/DeveloperDocumentation/PackagingGuides/bento4.md:64]:
```suggestion
Below is an example of how we can send a request to Bento4 to encrypt our content with Widevine and PlayReady DRM and package it:
```

**Review by PerritonBranch — CHANGES_REQUESTED** [review]

File: docs/source/DeveloperDocumentation/PackagingGuides/bento4.md (hunk 7 above)
Comment (bengo1990) [reply docs/source/DeveloperDocumentation/PackagingGuides/bento4.md:64]: Actioned.

File: docs/source/DeveloperDocumentation/PackagingGuides/bento4.md (hunk 4 above)
Comment (bengo1990) [reply docs/source/DeveloperDocumentation/PackagingGuides/bento4.md:15]: Actioned.

File: docs/source/DeveloperDocumentation/PackagingGuides/bento4.md (hunk 5 above)
Comment (bengo1990) [reply docs/source/DeveloperDocumentation/PackagingGuides/bento4.md:17]: Actioned.

File: docs/source/DeveloperDocumentation/PackagingGuides/bento4.md (hunk 6 above)
Comment (bengo1990) [reply docs/source/DeveloperDocumentation/PackagingGuides/bento4.md:60]: Actioned.

**Review by PerritonBranch — APPROVED** [review]

**Review by somerobmitchell — APPROVED** [review]

**Pull request merged**